src/features/settings/screens/UsersScreen/components/UserRoleModal/UserRoleModal.tsx

```diff
@@ -3,7 +3,8 @@ import React, { memo, useEffect } from "react";
 import { Col, Form, Input } from "antd";
```

Comment:
can you remove the Form from here and import what you need directly from the Form atom?